pegasus/sites.v3/code.org/public/private/k5survey/index.haml

```diff
@@ -117,34 +71,42 @@
       end.compact
     end.compact.flatten
   end
-
+%link(rel='stylesheet' type='text/css' href='/css/learn_tabs.css')
 %h1
   K5 Survey Statistics
-
+#learn-tabs{style: "padding-top:20px; padding-left:20px; padding-right:20px; background-color:white; background-color: rgb(219, 219, 219); overflow: hidden; margin-bottom: 10px;"}
+  %ul.nav.nav-tabs{style: "float: left;"}
+    %li.active.tab
+      %a{href:"/"} Results
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace